package.json

```
@@ -43,9 +43,9 @@
     "cp-package-json": "cp package.json ./src",
     "test": "npx ts-mocha test/{,**}/*.spec.ts",
     "build": "npm run cp-package-json && npx tsc",
-    "start": "node dist/index.js",
+    "start": "node src/index.ts",
```

Do you really want to change this? I think index.js is still the build output from 'npm run build'

---

Confirmed that this should stay as-is
```suggestion
    "start": "node dist/index.ts",
```

---

awesome - thanks for catching this!